src/ui/libs/DatalensChartkit/ChartKit/components/Widget/components/Table/WrappedHTMLNode.tsx

```diff
@@ -10,6 +11,14 @@ type Props<T extends React.ElementType> = {
     className?: string;
 };
 
+export function isWrappedHTML(value: unknown): value is WrappedHTML {
+    if (!value || typeof value !== 'object') {
+        return false;
+    }
+
+    return Object.keys(value).length === 1 && WRAPPED_HTML_KEY in value;
```

Comment:
Is it possible that checking for the number of keys is unnecessary? I can imagine that in the future there will be an additional parameter for generating html, for example

Reply:
Sure, lets remove this condition